CHANGELOG.md

```diff
@@ -1120,28 +1120,23 @@ activity and on its child transactions (which can be actuals, refunds, and adjus
 - Add Level B activity comments to budget exports
 - Strip leading/trailing whitespace and line breaks from comments in reports and exports
 
-## [unreleased]
+## [release-122]
 
 - Update seeds and create data migration for adding ISPF fund entity to the service
-- Hide ISPF reports when the ISPF feature flag is enabled
-- ISPF programmes cannot be created when the feature flag is enabled
```

Comment:
"ISPF programmes cannot be created when the feature flag is enabled" might not be covered explicitly now, but I think it's implied by "Hide ISPF-related entities in the service and exports when the feature flag is enabled"

Reply:
Do you think "Hide ISPF-facing functionality in the service and exports when the feature flag is enabled" would be explicit enough?